---
VLP16_DiagnosticsMessage  msg
